src/app/ApplicationTemplate.Shared/Presentation/Diagnostics/DiagnosticsOverlayViewModel.cs

```diff
@@ -19,7 +19,9 @@ public partial class DiagnosticsOverlayViewModel : ViewModel
 
 		public IDynamicCommand NavigateToDiagnosticsPage => this.GetCommandFromTask(async ct =>
 		{
+#if !PRODUCTION
```

Comment:
Have you tried to generate the with your changes. I don't think it works because you forgot to put the `//+:cnd:noEmit`. Also, you should use configuration for this, not compiler pre-conditions.

Reply:
https://github.com/nventive/UnoApplicationTemplate/blob/master/doc/Environments.md#compile-time-environments

Reply:
as an example: https://github.com/nventive/UnoApplicationTemplate/pull/19